Rlog/View/Schedule/ScheduleCreateView.swift

```diff
@@ -12,18 +12,21 @@ struct ScheduleCreateView: View {
     @Environment(\.dismiss) var dismiss
     
     var body: some View {
-        VStack(alignment: .leading, spacing: 0) {
-            workspace
-                .padding(.top, 40)
-            workDate
-                .padding(.top)
-            schedule
-                .padding(.top)
-            reasonInput
-                .padding(.top)
-            Spacer()
+        ScrollView {
+            VStack(alignment: .leading, spacing: 0) {
+                workspace
+                    .padding(.top, 40)
+                workDate
+                    .padding(.top)
+                schedule
+                    .padding(.top)
+                reasonInput
+                    .padding(.top)
+                Spacer()
```

Comment:
```suggestion
            VStack(alignment: .leading, spacing: 16.0) {
                workspace
                    .padding(.top, 40)
                workDate
                schedule
                reasonInput
                Spacer()
```

호옥시 Spacing을 활용해보면 어떠세요 ?

Reply:
Update뷰에서는 spacing으로 했었군요...?
수정 완료했습니다!